libwebauthn/src/transport/cable/qr_code_device.rs

```diff
@@ -185,50 +184,48 @@ impl CableQrCodeDevice<'_> {
     }
 
     async fn await_advertisement(&self) -> Result<(FidoDevice, DecryptedAdvert), Error> {
-        btleplug::manager::start_discovery(&[
-            Uuid::parse_str(CABLE_UUID_FIDO).unwrap(),
+        let stream = btleplug::manager::start_discovery_for_service_data(vec![
             Uuid::parse_str(CABLE_UUID_GOOGLE).unwrap(),
+            Uuid::parse_str(CABLE_UUID_FIDO).unwrap(),
```

Comment:
Is the order of the UUIDs here important? If so, maybe add a comment to avoid accidental reordering.

Reply:
Good point. The order is not important here, the Google UUID is only for caBLE v2 backwards compatibility. I've added some comments explaining this.